GeeksCoreLibrary/Components/Account/Services/AccountsService.cs

```diff
@@ -447,5 +454,199 @@ public async Task<List<RoleModel>> GetUserRolesAsync(ulong userId, bool includeP
             // Filter the roles based on the user's role IDs and return a List of the remaining rows.
             return roles.Where(role => userRoleIds.Contains(role.Id)).ToList();
         }
+
+        /// <summary>
+        /// NOTE: This method was moved from Accounts component and it should NOT be exposed to users.
+        /// Automatically logs in the user via ID. This function should never be made available publicly, only for internal usage to login after creating an account for example.
+        /// </summary>
+        /// <param name="userId">The ID of the user to login.</param>
+        /// <param name="mainUserId">The ID of the main user, if the user is logging in with a sub account.</param>
+        /// <param name="role">Used to set a custom role for the user separate of the Wiser role system</param>
+        /// <param name="extraDataForReplacements"></param>
+        /// <param name="settings"></param>
```

Comment:
The summary can be replaced with `/// <inheritdoc />`, then the summary of the interface is shown so we only need to manage this on one place.